.github/workflows/python.yml

```diff
@@ -0,0 +1,30 @@
+name: Python package CI
+
+on: [ push, pull_request ]
+
+jobs:
+  test:
```

Comment:
Assuming this action is just for `ruff` let's name this `ruff` or `ruff-lint` or something analogous instead of `test`.

Reply:
`python.yml` sounds fine to me, to be honest